source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -245,6 +301,11 @@ class WordDocument(UIADocumentWithTableNavigation,WordDocumentNode,WordDocumentB
 	shouldCreateTreeInterceptor=False
 	announceEntireNewLine=True
 
+	def _get_name(self):
+		# Microsoft Word duplicates the full title of the document on this control, which is redundant as it appears in the title of the app itself.
+		# Translators: The name of a control that shows a Microsoft Word document.
+		return _("Microsoft Word")

```

Comment:
Should this really be translatable as it is the name of an application? Personally, I'd vote for removing the name altogether, but that's just my personal preference.